app/enterprise/changelog.md

```diff
@@ -3,6 +3,156 @@ title: Kong Gateway (Enterprise) Changelog
+  the `any` option ensures that `luasec` will negotiate the most secure protocol available.
+
+#### PDK
+- This release includes a new JavaScript Plugin Development Kit (PDK). This addition
+  allows users to write Kong plugins in JavaScript and TypeScript.
+- This release includes support for the Protobuf plugin communication protocol, which can be used in
+  place of MessagePack to communicate with non-Lua plugins. [6941](https://github.com/Kong/kong/pull/6941)
+- This release enables the `ssl_certificate` phase on plugins in the `stream` module.
+  [6873](https://github.com/Kong/kong/pull/6873)
+
+#### Plugins
+- **New plugin:** [OPA](/hub/kong-inc/opa) (`opa`)
+  - The OPA plugin forwards requests to an Open Policy Agent and processes the request
+    only if the authorization policy allows for it. **Released as BETA and should not be deployed
+    in production environment.**
+- **New plugin:** Mocking (`mocking`)
+  - The Kong Mocking plugin leverages standards-based Open API Specifications (OAS)
+    for sending out mock responses to APIs. **Released as BETA and should not be deployed
+    in production environment.**
+- [Zipkin](/hub/kong-inc/zipkin) (`zipkin`)
+  - The plugin now supports OT and Jaeger style `uber-trace-id` headers. See `config.header_type` in
+    the [Parameters](/hub/kong-inc/zipkin/#parameters) section of the Zipkin
+    plugin documentation for more information. [101](https://github.com/Kong/kong-plugin-zipkin/pull/101 )
+  - The plugin now allows insertion of custom tags on the Zipkin request trace. See `config.tags_header`
+    in the [Parameters](/hub/kong-inc/zipkin/#parameters) section of the Zipkin
+    plugin documentation for more information. [102](https://github.com/Kong/kong-plugin-zipkin/pull/102)
+  - The plugin now allows the creation of baggage items on child spans.
+    [98](https://github.com/Kong/kong-plugin-zipkin/pull/98)
+- [JWT](/hub/kong-inc/jwt) (`jwt`) 
+  - The plugin now supports ES384 JWTs signature validation. [6854](https://github.com/Kong/kong/pull/6854)
+- Logging plugins: [File Log](/hub/kong-inc/file-log), [Loggly](/hub/kong-inc/loggly),
+  [Syslog](/hub/kong-inc/syslog), [TCP Log](/hub/kong-inc/tcp-log), [UDP Log](/hub/kong-inc/udp-log),
+  and [HTTP Log](/hub/kong-inc/http-log)
+  - Kong administrators now have a powerful new capability to transform logs to any format that’s needed
+    for capturing, indexing, and correlating logs. When formatting, developers also have the ability to
+    execute custom Lua code, opening up a new range of possibilities for generating logs to whichever specification
+    administrators need the most. This feature uses the new PDK method `kong.log.set_serialize_value`,
+    as well as the new sandbox capability, both introduced in Kong v2.3. [6944](https://github.com/Kong/kong/pull/6944)
+
+### Fixes
+
+#### Core
+- Topological sort now prioritizes core, avoiding problems when plugin entities use core entities but
```

Comment:
So this one is related to the "Migrations order is now guaranteed . . ." When I asked @kikito in the Google doc to explain the significance to the user he told me this, "Context: the only affected ones would be those who had plugins with migrations which depended on other plugins with migrations (so one had to run after the other). Usually our users don't setup their plugins like that - in part because, if they did, the random order in which migrations run before would give them trouble."